modules/luxonFile.js

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-class-assign */
+/* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-redeclare */
+/* eslint-disable no-use-before-define */
+/* eslint-disable new-cap */
+/* eslint-disable no-unreachable */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-mixed-operators */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-var */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable eqeqeq */
+/* eslint-disable no-sequences */
+/* eslint-disable no-return-assign */
+/* eslint-disable max-len */
+/* eslint-disable consistent-return */
+/* eslint-disable no-void */
+/* eslint-disable no-cond-assign */
+/* eslint-disable no-loop-func */
+/* eslint-disable no-useless-escape */
+/* eslint-disable prefer-rest-params */
```

Comment:
- [ ] - [ ] Same here, Please remove disable lines for `eslint` and You can ignore the whole file from eslint check. see this [link](https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file) for more info about how to do it.